trestle/core/generators.py

```diff
@@ -80,8 +80,10 @@ def generate_sample_value_by_type(
             return 'REPLACE_ME'
     elif hasattr(type_, '__name__') and 'ConstrainedIntValue' in type_.__name__:
         # create an int value as close to the floor as possible does not test upper bound
-        multiple = type_.multiple_of or 1  # default to every integer
-        floor = type_.ge or type_.gt + 1 or 0  # default to 0
+        multiple = type_.multiple_of if type_.multiple_of else 1  # default to every integer
```

Comment:
Looks like conint is only used by protocol via portrange

Reply:
I guess we need to support this irrespective given the complexity of NIST's changes.